.github/workflows/release.yaml

```diff
@@ -2,7 +2,7 @@ name: Publish snap
 
 on:
   push:
-    branches: [ main ]
+    branches: [ '*.x' ]
```

Comment:
Why use `*.x` instead of more explicit `3.x` ?

Reply:
Good point, we can use `3.x` directly.